tests/Unit/Domain/Creators/Test_Interval.cpp

```diff
@@ -83,80 +90,236 @@ void test_interval() {
+                                               upper_bound,
+                                               refinement_level[0],
+                                               grid_points[0],
+                                               std::array<bool, 1>{{true}},
+                                               nullptr};
+    test_interval_construction(
+        periodic_interval, lower_bound, upper_bound, grid_points,
+        refinement_level,
+        std::vector<DirectionMap<1, BlockNeighbor<1>>>{
+            {{Direction<1>::lower_xi(), {0, aligned_orientation}},
+             {Direction<1>::upper_xi(), {0, aligned_orientation}}}},
+        std::vector<std::unordered_set<Direction<1>>>{{}}, std::tuple<>{}, {},
+        {});
+  }
+  {
+    INFO("With boundary condition");
+    std::vector<DirectionMap<
+        1, std::unique_ptr<domain::BoundaryConditions::BoundaryCondition>>>
+        expected_boundary_conditions{1};
+    expected_boundary_conditions[0][Direction<1>::lower_xi()] =
+        std::make_unique<
+            TestHelpers::domain::BoundaryConditions::TestBoundaryCondition<1>>(
+            Direction<1>::lower_xi(), 0);
+    expected_boundary_conditions[0][Direction<1>::upper_xi()] =
+        std::make_unique<
+            TestHelpers::domain::BoundaryConditions::TestBoundaryCondition<1>>(
+            Direction<1>::upper_xi(), 0);
+    const creators::Interval interval{
+        lower_bound,
+        upper_bound,
+        refinement_level[0],
+        grid_points[0],
+        expected_boundary_conditions[0][Direction<1>::lower_xi()]->get_clone(),
+        expected_boundary_conditions[0][Direction<1>::upper_xi()]->get_clone(),
+        nullptr};
+    test_interval_construction(
+        interval, lower_bound, upper_bound, grid_points, refinement_level,
+        std::vector<DirectionMap<1, BlockNeighbor<1>>>{{}},
+        std::vector<std::unordered_set<Direction<1>>>{
+            {{Direction<1>::lower_xi()}, {Direction<1>::upper_xi()}}},
+        std::tuple<>{}, {}, expected_boundary_conditions);
 
-  const creators::Interval periodic_interval{
-      lower_bound, upper_bound, std::array<bool, 1>{{true}},
-      refinement_level[0], grid_points[0]};
-  test_interval_construction(
-      periodic_interval, lower_bound, upper_bound, grid_points,
-      refinement_level,
-      std::vector<DirectionMap<1, BlockNeighbor<1>>>{
-          {{Direction<1>::lower_xi(), {0, aligned_orientation}},
-           {Direction<1>::upper_xi(), {0, aligned_orientation}}}},
-      std::vector<std::unordered_set<Direction<1>>>{{}}, std::tuple<>{}, {});
+    const std::unique_ptr<domain::BoundaryConditions::BoundaryCondition>
+        periodic = std::make_unique<TestHelpers::domain::BoundaryConditions::
+                                        TestPeriodicBoundaryCondition<1>>();
+    const creators::Interval periodic_interval{
+        lower_bound,    upper_bound,           refinement_level[0],
```

Comment:
If you are referring to the large spaces, this is clang-format lining up the commas. If you mean something else, I'm not sure what since running clang-format over the entire file doesn't change anything

Reply:
If this is the output of clang-format, then so be it... though I've never so strongly disagreed with it

Reply:
And I have yet to figure out when it decides this is the "right" formatting vs. just making a long list...